### ISSUE TITLE
Could not start docker compose due to configuration issue

### ISSUE THREAD

**Issue by lephuocmy668** [stellar/java-stellar-anchor-sdk#881]
### What version are you using?
stellar/anchor-platform:2.0.0-rc.2

### What did you do?
I tried to start docker compose service follow [this document](https://developers.stellar.org/docs/anchoring-assets/anchor-platform/installation) with config bellow but received an error:
```
version: "3.8"
services:
  platform-server:
    image: stellar/anchor-platform:2.0.0-rc.2
    command: --sep-server
    ports:
      - "8080:8080"
    env_file:
      - ./dev.env
    volumes:
      - ./config:/home
  db:
    image: postgres:14
    ports:
      - "5432:5432"
    env_file:
      - ./dev.env
```

### What happened?
I have been receiving this error:
```
platform-server_1  | java.io.IOException: Failed to read configuration from /home/config/dev.services.yaml
platform-server_1  |    at org.stellar.anchor.platform.configurator.ConfigManager.getConfigFileAsResource(ConfigManager.java:147) ~[platform-2.0.0-rc.2-plain.jar!/:2.0.0-rc.2]
platform-server_1  |    at org.stellar.anchor.platform.configurator.ConfigManager.processConfigurations(ConfigManager.java:104) ~[platform-2.0.0-rc.2-plain.jar!/:2.0.0-rc.2]
platform-server_1  |    at org.stellar.anchor.platform.configurator.SepConfigManager.initialize(SepConfigManager.java:25) ~[platform-2.0.0-rc.2-plain.jar!/:2.0.0-rc.2]
platform-server_1  |    at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:604) ~[spring-boot-2.7.9.jar!/:2.7.9]
platform-server_1  |    at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:373) ~[spring-boot-2.7.9.jar!/:2.7.9]
platform-server_1  |    at org.springframework.boot.SpringApplication.run(SpringApplication.java:306) ~[spring-boot-2.7.9.jar!/:2.7.9]
platform-server_1  |    at org.stellar.anchor.platform.AnchorPlatformServer.start(AnchorPlatformServer.java:43) ~[platform-2.0.0-rc.2-plain.jar!/:?]
platform-server_1  |    at org.stellar.anchor.platform.ServiceRunner.startSepServer(ServiceRunner.java:70) ~[classes!/:?]
platform-server_1  |    at org.stellar.anchor.platform.ServiceRunner.main(ServiceRunner.java:32) ~[classes!/:?]
platform-server_1  |    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
platform-server_1  |    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
platform-server_1  |    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
```

I checked the configuration file in docker container, `/home/config/dev.services.yaml` is existing:
![Screen Shot 2023-05-15 at 09 30 23](https://github.com/stellar/java-stellar-anchor-sdk/assets/18652812/ee5cf9a3-7359-4813-947e-ecd4ff3b5473)

### What did you expect to see instead?
Docker compose services should be started successfully.

**Comment by Ifropc** [comment]
Hi @lephuocmy668 ,
Thanks for using the anchor platform.
It looks like in your .env file you specified path as `/home/config/dev.services.yaml` while the correct path seems to be `/home/dev.services.yaml`

**Comment by dhesetty** [comment]
Hi @lephuocmy668 ,
I am also facing same issue, got any resolution?

**Comment by Ifropc** [comment]
Should be fixed in #890 and will be available in the latest `stellar/anchor-platform:edge` image
